bucket/linkerd.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://linkerd.io/",
+    "description": "Linkerd is an ultralight service mesh for Kubernetes. It gives you observability, reliability, and security without requiring any code changes.",
+    "license": "Apache-2.0",
+    "version": "2.3.2",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/linkerd/linkerd2/releases/download/stable-2.3.2/linkerd2-cli-stable-2.3.2-windows.exe#linkerd.exe",
+            "hash": "9f35d81cf178bb4f2babc5cb0b05a4a1775932bfb1b399214b52b6aae85a44e4",
+        }
+    },
+    "bin": "linkerd.exe",
+    "checkver": {
+        "github": "https://github.com/linkerd/linkerd2/"
```

Comment:
This will not work since latest release point to some edge release and their tags are using `stable-` prefix. 

Latest stable is 6th release from top

Reply:
@Ash258 is there any way to filter based on regex?

Reply:
Would this work?

```
"checkver": {
  "url": "https://github.com/linkerd/linkerd2/releases",
  "re": "stable-(?<version>[\\d.]+)"
},
```

Reply:
```json
    "checkver": {
        "url": "https://github.com/linkerd/linkerd2/releases",
        "regex": "tag/stable-([\\d.]+)"
    },
```

Reply:
Ah thx! Should be fixed now. :)